Dockerfile.lab

```diff
@@ -28,17 +28,23 @@ ENV INCORE=${INCORE} \
     PYINCORE_VIZ=${PYINCORE_VIZ} \
     PYINCORE_DATA=${PYINCORE_DATA}
 
+# Set conda channels
```

Comment:
can this be combined so we don't end up with 2 layers in the final docker image.